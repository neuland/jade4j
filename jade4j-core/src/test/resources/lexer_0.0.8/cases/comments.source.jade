//-
  s/s.

//- test/cases/comments.source.jade

//-
  test/cases/comments.source.jade
  when
  ()
