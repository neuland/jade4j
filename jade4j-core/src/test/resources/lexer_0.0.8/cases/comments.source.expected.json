{"type":"comment","line":1,"val":"","buffer":false}
{"type":"start-pipeless-text","line":1}
{"type":"text","line":2,"val":"s/s."}
{"type":"newline","line":3}
{"type":"text","line":3,"val":""}
{"type":"end-pipeless-text","line":3}
{"type":"newline","line":4}
{"type":"comment","line":4,"val":" test/cases/comments.source.jade","buffer":false}
{"type":"text","line":5,"val":""}
{"type":"newline","line":6}
{"type":"comment","line":6,"val":"","buffer":false}
{"type":"start-pipeless-text","line":6}
{"type":"text","line":7,"val":"test/cases/comments.source.jade"}
{"type":"newline","line":8}
{"type":"text","line":8,"val":"when"}
{"type":"newline","line":9}
{"type":"text","line":9,"val":"()"}
{"type":"end-pipeless-text","line":9}
{"type":"eos","line":9}